store/streaming/README.md

```diff
@@ -0,0 +1,64 @@
+# State Streaming Service
+This package contains the constructors for the `StreamingService`s used to write state changes out from individual KVStores to a
+file or stream, as described in [ADR-038](../docs/architecture/adr-038-state-listening.md) and defined in [types/streaming.go](../types/streaming.go).
+The child directories contain the implementations for specific output destinations.
+
+Currently, a `StreamingService` implementation that writes state changes out to files is supported, in the future support for additional
+output destinations can be added.
+
+The `StreamingService` is configured from within an App using the `AppOptions` loaded from the app.toml file:
+
+```toml
+[store]
+    streamers = [ # if len(streamers) > 0 we are streaming
+        "file", # name of the streaming service, used by constructor
+    ]
+
+[streamers]
+    [streamers.file]
+        keys = ["list", "of", "store", "keys", "we", "want", "to", "expose", "for", "this", "streaming", "service"]
+        writeDir = "path to the write directory"
+        prefix = "optional prefix to prepend to the generated file names"
+```
+
+`store.streamers` contains a list of the names of the `StreamingService` implementations to employ which are used by `NewServiceConstructor`
+to return the `ServiceConstructor` for that particular implementation:
+
+ 
+```go
+listeners := cast.ToStringSlice(appOpts.Get("store.streamers"))
```

Comment:
based on the toml example above, it would be just `streamers`, not `store. streamers`.

Reply:
In this instance it is loading the `store.streamers` array to first get the list of streamer names, inside `NewServiceConstructor` this name is used to select the correct service constructor and that constructor loads the service-specific parameters from the `streamers.{streamer_name}` TOML mapping.